client/index.html

```diff
@@ -9,23 +9,23 @@
 	<link rel="preload" as="script" href="js/bundle.js">
 	<link rel="stylesheet" href="css/bootstrap.css">
 	<link rel="stylesheet" href="css/style.css">
-	<link id="theme" rel="stylesheet" href="{{ theme }}">
+	<link id="theme" rel="stylesheet" href="<%- theme %>">
```

Comment:
Why do we go back to Lodash templates? It was overall a good thing that we switch from that to Handlebars, since we use Handlebars for anything else 😕

Reply:
Since most templates moved to client, they are still being compiled by handlebars. But this leaves just a couple of variables that need to be set in index.html, and it felt like a good idea to drop a somewhat heavy runtime dependency from the server.

Reply:
> it felt like a good idea to drop a somewhat heavy runtime dependency from the server.

Yep, absolutely!

Overall, shouldn't we be serving a totally static `index.html` and all values would be set by the client (using that same `getClientConfiguration`)? That would solve this along with a few other things, no?

Reply:
It would be nice, yes, but these variables that I kept have to be set before JS loads (especially `transports` which is used for connecting socket.io), and the initial delay of loading the theme.